CryptoPrices/Sources/Data/Sources/NetworkExtension/Parameters/GetChartPricesParameters.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  GetChartPricesParameters.swift
+//  Data
+//
+//  Created by Minh Pham on 27/12/2022.
+//
+
+public struct GetChartPricesParameters: Parameterable {
+
+    public let id: String
+    public let vsCurrency = "usd"
+    public let days: String
```

Comment:
The `days`'s type should be `Int` 🙏

Reply:
I am not sure why you raised this but the `day`'s type from the Coingecko [document](https://www.coingecko.com/en/api/documentation) is `String`, not `Int` 😅

![Screenshot 2022-12-28 at 15 14 46](https://user-images.githubusercontent.com/70877098/209780905-502b2c5a-bf08-422c-b4bd-7688ac910226.png)

Reply:
Oh, I missed this part 🙏

But if you really think about it, technically `String` can be used to represent _number of days_, but semantically `Int` would be the better choice. I don't know the intention of the CoinGecko here. The other API like `coins/market` has the parameters with the type of `Int`:

<img width="572" alt="Screenshot 2022-12-29 at 09 15 43" src="https://user-images.githubusercontent.com/17972674/209894597-733826d7-175d-43cb-a633-01b105818974.png">

They could've used `String` for `per_page` and `page` here as they did for `days` but they didn't. My best guess is this has something to do with the types they use in their database schema, which may have some history 😄 

Put that aside, speaking of implementation, either `Int` or `String` works cause we use string interpolation when encoding the parameters (i.e. `"\(days)"`. But it's fair and practically correct to use `String` here to match and keep it in sync with the CoinGecko API 🙏

Reply:
[Updated] Now I realized the `days` parameter may accept a `"max"` value, which may cap the maximum ranges that they can return.  Finally `String` makes sense to me 😄

Reply:
@Thieurom yeah, you are totally right that this `day`'s type should be `Int` and it would definitely make more sense than `String` here. Unfortunately, CoinGecko didn't do it for this API 🙈

So I will have to blindly stick with their API interface, what a pain 🤷‍♂️

Reply:
> [Updated] Now I realized the `days` parameter may accept a `"max"` value, which may cap the maximum ranges that they can return.  Finally `String` makes sense to me 😄 

Oh okay, the API also supports text constraint, got it now 😂